src/components/Ad.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Grid } from "semantic-ui-react";
+import mercedesImg from "../images/mercedesAd.jpg";
+import lagavulinImg from "../images/lagavulinAd.jpg";
+
+const Ad = ({ type }) => {
+  const adList = [
+    {
+      id: "ad-1",
+      href: "https://www.mercedes-benz.com/en/",
```

Comment:
pass in this as props instead

Reply:
Would reading an external json work fine? Would make editing it easier at least for owner. We are selecting from the list using a prop right now.